src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,19 @@
+package calculator;
+
+public record StringCalculator(
+    InputValidator inputValidator,
+    DelimiterParser delimiterParser,
+    NumberParser numberParser,
+    SumCalculator sumCalculator
+) {
+
+  public int add(String input) {
```

Comment:
들여쓰기가 클래스별로 '2 spaces'와 '4 spaces'가 혼용되어 있는 것 같아요.
소스코드 전체적으로 한번 점검 부탁드릴께요.
> 들여쓰기는 '2 spaces'가 아닌 '4 spaces'로 한다.
```suggestion
    public int add(String input) {
```